util/src/main/java/io/grpc/util/AdvancedTlsX509KeyManager.java

```diff
@@ -131,14 +129,17 @@ public Closeable updateIdentityCredentialsFromFile(File keyFile, File certFile,
       throw new GeneralSecurityException(
           "Files were unmodified before their initial update. Probably a bug.");
     }
+    if (checkNotNull(unit, "unit").toMinutes(period) < MINIMUM_REFRESH_PERIOD) {
+      log.log(Level.INFO, "Provided refresh period of {} {} is too small. "
+          + "Default value of {} minute(s) will be used.", new Object[] {period, unit.name(),
```

Comment:
Since JUL is used, most of the times it's configured to print method as well, i.e.
`io.grpc.util.AdvancedTlsX509KeyManager updateIdentityCredentialsFromFile
INFO: Provided refresh period...`
IMHO it's already specific enough for a user to understand the issue. WDYT?

Reply:
SGTM.